### PR TITLE
Fix saving attachments on mobile

### PR DESCRIPTION
**Pull request by MarcoPolo**
@keybase/react-hackers 

Makes message details work the same for attachments popups and normal messages.

### CODE REVIEW

**Comment by cjb** [comment]
@MarcoPolo I tried the branch and got "Attempt to perform routeTree: navigateAppend on /tabs:chatTab/.../attachment/messageAction .. Invalid route child: messageAction.

**Comment by cjb** [comment]
Oh, perhaps I double-tapped the three dots in the bottom left of the image.

**Comment by cjb** [comment]
I clicked Share Image and I think the app crashed.

**Comment by cjb** [comment]
Got the same invalid route child double tapping on the image itself in the conversation view.

**Comment by MarcoPolo** [comment]
hmm this actually breaks details in popup on desktop. will fix

**Comment by MarcoPolo** [comment]
fixed desktop

File: shared/chat/conversation/attachment-popup/index.native.js [hunk 1]
```diff
@@ -46,18 +45,9 @@ const AttachmentPopup = ({message, detailsPopupShowing, isZoomed, onCloseDetails
           <Text type='BodySmall' style={{color: globalColors.black, marginTop: globalMargins.large}}>Your device can not preview this file.</Text>
         </Box>
         <Box style={styleHeaderFooter}>
-          <Icon type='iconfont-ellipsis' onClick={detailsPopupShowing ? onCloseDetailsPopup : onOpenDetailsPopup} />
```
Comment (chromakode) [inline shared/chat/conversation/attachment-popup/index.native.js]: Can you now delete the `onCloseDetailsPopup`/`onOpenDetailsPopup` stuff in `attachment-popup/container.js`?

**Pull request merged**